libraries/AP_Notify/Display.cpp

```diff
@@ -536,40 +536,42 @@ void Display::update_text_empty(uint8_t r)
 {
     char msg [DISPLAY_MESSAGE_SIZE] = {};
     memset(msg, ' ', sizeof(msg)-1);
-    _movedelay = 4;
+    _movedelay = 0;
     _mstartpos = 0;
     draw_text(COLUMN(0), ROW(r), msg);
 }
 
 void Display::update_text(uint8_t r)
 {
-    char msg [DISPLAY_MESSAGE_SIZE] = {0};
-    char txt [NOTIFY_TEXT_BUFFER_SIZE] = {0};
+    char msg [DISPLAY_MESSAGE_SIZE] = {};
```

Comment:
@muramura My reasoning here is that `{0}` is just one of many special cases you can have when initialising an array.  You could have `{0, 7, 3}` or `{2,1,4}`.

The standard just says everything that isn't nominated is zeroed, so we may as well not nominate anything.  Here we might save *two bytes*!  I'm banking those.